### ISSUE TITLE
How standing and sitting in long trial being recorded?

### ISSUE THREAD

**Issue by HosseinShahverdi** [mmalekzadeh/motion-sense#10]
Thanks for your valuable dataset. I was wondering in what condition performers record data for long duration for standing and sitting activities? did they just stand/sit in/on a place?

**Comment by mmalekzadeh** [comment]
Yes. They were sitting or standing in one place and they were asked to do it as natural as they could.
